### PR TITLE
Adding babel support and hooking into build system

### PR DESCRIPTION
**Pull request by ecwillis**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -10,16 +10,22 @@ const config = require('confi')({
...
 mkdirp.sync(path.join(config.CWD, '.dist'));
 
 const watchedFiles = [
   path.join(__dirname, 'styles', '**/*.css'),
+  path.join(__dirname, 'scripts', '**/*.js'),
 ];
 const stylesheets = Object.keys(config.stylesheets);
+const scripts = Object.keys(config.scripts);
 
 const watcher = chokidar.watch(watchedFiles, {
```
Comment (jgallen23) [inline app.js]: should we set up separate watchers for css and js?

File: app.js [hunk 2]
```diff
@@ -10,16 +10,22 @@ const config = require('confi')({
...
+const jsProcessor = require('./tasks/script.js');
+
 const mkdirp = require('mkdirp');
 const debounce = require('lodash.debounce');
 
 mkdirp.sync(path.join(config.CWD, '.dist'));
 
 const watchedFiles = [
   path.join(__dirname, 'styles', '**/*.css'),
+  path.join(__dirname, 'scripts', '**/*.js'),
```
Comment (jgallen23) [inline app.js]: we should probably watch the client kit directory for yaml changes too, right?

Comment (ecwillis) [inline app.js]: Yeah - probably on both styles and scripts?

File: app.js (hunk 1 above)
Comment (ecwillis) [inline app.js]: We probably should. @jgallen23

File: scripts/default.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+'use strict';
```
Comment (jgallen23) [inline scripts/default.js]: what is this?

Comment (jgallen23) [inline scripts/default.js]: maybe move to an example directory or somethin?

File: app.js (hunk 2 above)
Comment (dawnerd) [inline app.js]: unrelated to babel but on helpers as well

File: scripts/default.js (hunk 3 above)
Comment (ecwillis) [inline scripts/default.js]: Yeah. just something to play with... Do we have anything to start with script wise?

Comment (dawnerd) [inline scripts/default.js]: domodule?

**Comment by dawnerd** [comment]
Looks good to me

**Pull request merged**